components/nimbus/src/persistence.rs

```diff
@@ -442,6 +604,668 @@ mod tests {
+        vec![
+            json!({
+                "schemaVersion": "1.0.0",
+                "slug": "secure-gold", // change when copy/pasting to make experiments
+                "endDate": null,
+                "featureIds": ["abc"], // change when copy/pasting to make experiments
+                "branches":[
+                    {
+                        "slug": "control",
+                        "ratio": 1,
+                        "feature": {
+                            "featureId": "abc", // change when copy/pasting to make experiments
+                            "enabled": false,
+                            "value": {"color": "green"}
+                        }
+                    },
+                    {
+                        "slug": "treatment",
+                        "ratio":1,
+                        "feature": {
+                            "featureId": "abc", // change when copy/pasting to make experiments
+                            "enabled": true,
+                            "value": {}
+                        }
+                    }
+                ],
+                "channel": "nightly",
+                "probeSets":[],
+                "startDate":null,
+                "appName": "fenix",
+                "appId": "org.mozilla.fenix",
+                "bucketConfig":{
+                    // Setup to enroll everyone by default.
+                    "count":10_000,
+                    "start":0,
+                    "total":10_000,
+                    "namespace":"secure-gold", // change when copy/pasting to make experiments
+                    "randomizationUnit":"nimbus_id"
+                },
+                "userFacingName":"Diagnostic test experiment",
+                "referenceBranch":"control",
+                "isEnrollmentPaused":false,
+                "proposedDuration": 21,
+                "proposedEnrollment":7,
+                "targeting": "true",
+                "userFacingDescription":"This is a test experiment for diagnostic purposes.",
+            }),
+            json!({
+                "schemaVersion": "1.5.0",
+                "slug": "ppop-mobile-test",
+                // "arguments": {}, // DEPRECATED
+                // "application": "org.mozilla.firefox_beta", // DEPRECATED
+                "appName": "fenix",
+                "appId": "org.mozilla.firefox_beta",
+                "channel": "beta",
+                "userFacingName": "[ppop] Mobile test",
+                "userFacingDescription": "test",
+                "isEnrollmentPaused": false,
+                "bucketConfig": {
+                    "randomizationUnit": "nimbus_id",
+                    "namespace": "fenix-default-browser-4",
+                    "start": 0,
+                    "count": 10000,
+                    "total": 10000
+                },
+                "probeSets": [],
+                // "outcomes": [], NOT CURRENTLY (YET?) IMPLEMENTED
```

Comment:
outcomes is an analysis specific piece of the schema and not of interest to the SDK.

Reply:
OK, update the coment.